spec/controllers/collections_controller_spec.rb

```diff
@@ -94,7 +97,10 @@ def to_solr(solr_doc={})
       @asset1 = GenericFile.create!
       @asset2 = GenericFile.create!
       post :create, batch_document_ids: [@asset1,@asset2], collection: {title: "My Secong Collection ", description: "The Description\r\n\r\nand more"}
-      expect(assigns[:collection].members).to eq([@asset1,@asset2])
```

Comment:
If order matters, then something is going wrong.  These tests failed prior to my additional code.  They were revealed when I fixed the jetty jdk issue.  

How should we proceed?  Shall I remove the changes to the tests?  Will you accept the PR inspite of the failures since they exist already?